docs/quickstart.rst

```diff
@@ -0,0 +1,83 @@
+.. toctree::
+   :maxdepth: 1
+
+Quickstart
```

Comment:
reference the docker image if people want to start with a bare environment and not install into their local system?

Reply:
i will add this after we have a docker release procedure.